web_external/Literature/LiteratureView.js

```diff
@@ -0,0 +1,64 @@
+import _ from 'underscore';
+
+import View from '../view';
+
+import publicationsBibtex from './publications.bibtex';
+import LiteraturePageTemplate from './literaturePage.pug';
+import './literaturePage.styl';
+
+const LiteratureView = View.extend({
+    initialize: function (settings) {
+        this.publicationsJson = publicationsBibtex;
+
+        this.publicationsJson.sort((a, b) => {
+            const aYear = a.entryTags.year || 0;
+            const bYear = b.entryTags.year || 0;
+            if (aYear < bYear) {
+                return 1;
+            } else if (aYear > bYear) {
+                return -1;
+            } else {
+                const aAuthor = a.entryTags.author || '';
+                const bAuthor = b.entryTags.author || '';
+                return aAuthor > bAuthor ? 1 : -1;
+            }
+        });
```

Comment:
Can you add a short description of the comparator? E.g. "Sort BibTeX entries by year, in descending order, then by author.